### ISSUE TITLE
Crossing-training does not handle research when stopping

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#414]
```
>
Noting that your research is progressing nicely, you continue to focus on it.
>
Moon Mage Jairne just arrived.
>
Mooselurk gestures.
Mooselurk's cambrinth armband emits a loud *snap*.
>
[crossing-training]>tdp
You have 136 TDPs.
>
[crossing-training: ***Skills capped and researching***]
>
You continue to study Warding magic.
>
Tekronn seems to relax as he continues to play a spritely polka on his copper zills.
>
Noting that your research is progressing nicely, you continue to focus on it.
>
Tiny Dancer Mooselurk patters south.
>
[crossing-training]>tdp
You have 136 TDPs.
>
[crossing-training]>stop play
--- Lich: hunting-buddy active.
[hunting-buddy: 0:["common", "common-travel", "drinfomon"]]
[bootstrap: ["common", "common-travel", "drinfomon"]]
[bootstrap: common:["common-travel", "drinfomon"]]
[bootstrap: common-travel:["drinfomon"]]
[bootstrap: drinfomon:[]]
[hunting-buddy: waiting 0:["common", "common-travel", "drinfomon"]]
[hunting-buddy: done 0:["common", "common-travel", "drinfomon"]]
--- Lich: go2 active.
[go2: ETA: 0:00:11 (56 rooms to move through)]
[go2]>south
In the name of love?
>
--- Lich: crossing-training has exited.
```

Training-manager then started a hunting cycle. When combat-trainer started up:
```
>
You continue to study Warding magic.
>
[combat-trainer]>stand
Finishing your meditation, you feel that you can still gather a much bigger charge.
>
You stand back up.
>
[combat-trainer]>summon weapon  Small Edged
Your knowledge of weaponsmithing improves the pattern you are attempting to summon.
Closing your eyes, you grunt briefly in effort as you sense a small earthen ethereal fissure open in front of you.  Thrusting your hand through, you draw out a stone scimitar.  As you open your eyes, the fissure closes.
Roundtime: 1 sec.
>
[combat-trainer]>stance set 100 83 0
Setting your Evasion stance to 100%, your Parry stance to 83%, and your Shield stance to 0%.  You have 0 stance points left.
>
[combat-trainer: found buff missing: Ethereal Fissure]
[combat-trainer: prepare spell: {"abbrev"=>"ETF", "recast"=>-1, "mana"=>23, "cambrinth"=>[11, 11, 10], "expire"=>"fissure collapses in on itself, winking out of existence", "cast"=>"cast fire", "name"=>"Ethereal Fissure"}]
[combat-trainer]>pre ETF 23
Are you sure you want to do that?  You'll interrupt your research!
[Enter your command again if you want to.]
>
You continue to study Warding magic.
```

Crossing-training should either wait for research to finish or stop it.

**Comment by rpherbig** [comment]
Should be fixed by b750063f04c267892107d41a2207d98b13ba4720 - I will monitor for a few days

**Comment by rpherbig** [comment]
Looks good
